packages/playwright-core/src/server/android/android.ts

```diff
@@ -336,42 +335,59 @@ export class AndroidDevice extends SdkObject {
   }
 
   private async _refreshWebViews() {
-    const sockets = (await this._backend.runCommand(`shell:cat /proc/net/unix | grep webview_devtools_remote`)).toString().split('\n');
+    // possible socketName, eg: webview_devtools_remote_32327, webview_devtools_remote_32327_zeus, chrome_devtools_remote, content_shell_devtools_remote
```

Comment:
`chrome_devtools_remote` and `content_shell_devtools_remote` aren't web views though, why do we want to include them?

Reply:
I just expect to support more socketNames. 

Additionally. I cannot find  chromium webview app (package: org.chromium.webview_shell). I downloaded the app latest version from https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Android/ .  But it't chromium version is 77 which is not supported version  so that I cannot write and run test file.

Reply:
> I just expect to support more socketNames.

But where would they be coming from? It'd be nice to understand what you are trying to achieve.

Reply:
> But where would they be coming from? It'd be nice to understand what you are trying to achieve.

Sorry, I mean I just find the old version of chromium webview app, it is not work , it cannot run test files . So I use android chrome to test webview.

Reply:
> `chrome_devtools_remote` and `content_shell_devtools_remote` aren't web views though, why do we want to include them?

Yes, I agree with you. I already add the "webview" prefix. Please code review.